bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/Target.scala

```diff
@@ -48,6 +48,23 @@ object Target {
       PLens.nil.run
     ))
 
+  val pm: Target @?> ProperMotion =
+    PLens(_.fold(
+      PLens.nil.run,
+      SiderealTarget.pm.run,
+      PLens.nil.run,
+      PLens.nil.run,
+      PLens.nil.run
+    ))
```

Comment:
I think I could use a bit of schooling here.  What is going on in this code?  `PLens` is partial lens I take it.